IRSA-README.md

```diff
@@ -0,0 +1,88 @@
+well defined policies consistent with your Enterprise Security guidelines, please speak with your Domino Professional 
+Services contact for additional information, assistance and support.
+
+### When should you consider using IRSA with Domino
+
+  - Does your Enterprise Policy forbid the use of long lived credentials?
+    
+  - Would you like to transparently and securely propagate IAM Role to all Domino Workloads including a 
+[Domino Scheduled Job](https://docs.dominodatalab.com/en/latest/user_guide/5dce1f/schedule-jobs/) ?
+    
+  - Would you like to configure a Domino Workload with a set of IAM Roles based on a set of complex custom criteria?
+
+
+### Why does using IRSA with Domino help achieve the above goals?
+
+  - When using the IRSA support in Domino, any Domino hosted workloads can assume IAM roles in any AWS account configured to
+    trust the Domino Platform
+    
+  - IRSA support in Domino allows configuration of multiple roles for a Domino workload where the boto3 library can 
+    switch across the multiple profiles configured in the ~/.aws/config file
+    
+  - The mapping of a workload to a set of roles can be configured based on custom configurable rules. This allows for 
+    complex mappings to be supported
+
+### How does IRSA work with Domino?
+ 1 . For a Domino Workload (implemented as a K8s Pod) to be able to assume an IAM Role, it must use the 
+ `STS:AssumeRoleWithWebIdentity` API which returns temporary security credentials. But first the caller (in this case 
+ a pod) needs to be authenticated with a Web Identity Provider that is compatible with the OpenID connect (OIDC) protocol.
+
+ 2. Starting [1.12](https://kubernetes.io/docs/concepts/storage/projected-volumes/) Kubernetes allows an OIDC provider
+    to be associated with it. An EKS Cluster provides a unique OIDC provider bound to the `KubeAPIServer` server. A JWT 
+    created by this OIDC provider can be mounted as a projected volume in the Pod. This JWT is associated with the K8s 
+    Service Account associated with the workload. It is distinct from the JWT issued by the `KubeAPIServer`. This JWT 
+    acts as a workload identity credential that can be validated by AWS STS in order to pass IAM role credentials to 
+    that workload.
+
+3. This provider is configured as an IAM identity Provider within an AWS Account the EKS workloads want to assume an 
+   IAM Role. The key point here is that the EKS cluster and the IAM Role being assumed by a workload running on EKS do 
+   not need to be in the same AWS Account. 
+
+4. An AWS Admin will configure the IAM Role Trust Policy to enable a specific K8s Service Account (as identified by the
+   JWT above) to assume the Role. AWS STS will validate the Workload Identity and return temporary IAM Role Credentials
+   to a workload if an IAM Role Trust Policy allows it.
+
+5. Typically the amazon-eks-pod-identity-webhook will apply mutations based on annotations defined on the Service 
```

Comment:
Should we mention the aws webhook? Maybe we just say domino enables you to perform the same pattern on any non-aws kubernetes cluster 

Not a big deal, we can leave it in as well

Reply:
Good point - I rewrote


5. Domino adaptation of IRSA operates on a shared trust policy. A Domino mutating webhook framework called [Domsed](https://github.com/dominodatalab/domino-field-solutions-installations/tree/main/domsed) 
   configures a Domino workload definition to enable passing an identity issued by the OIDC Provider to IAM. We will refer to this identity as the Workload Identity. An AWS Administrator on the other hand configures a trust relationship between an IAM Role and this Workload Identity. This allows a Domino workload to assume one or more roles if those IAM Roles have their Trust Policy configured to permit the workload identity to assume them.
   
 We will discover in subsequent articles of this series, how this mechanism is cloud provider neutral and allows workloads in Domino  data planes running on-premise or in other cloud providers like GCP and Azure to assume AWS IAM Roles.